com.unity.multiplayer.mlapi/Tests/Runtime/NetworkTransformTests.cs

```diff
@@ -0,0 +1,151 @@
+using System;
+using System.Collections;
+using System.Text.RegularExpressions;
+using MLAPI.Prototyping;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using static MLAPI.Prototyping.NetworkTransform;
+
+namespace MLAPI.RuntimeTests
+{
+    [TestFixture(true)]
+    [TestFixture(false)]
```

Comment:
huh? what's this? which one should we delete here?

Reply:
delete?
Here's some doc on nunit's TestFixture, it's pretty great!
https://docs.nunit.org/articles/nunit/writing-tests/attributes/testfixture.html

Reply:
ah, those are 2 different tests (with different parameters), I see.
what do you think about doing something like this?
```suggestion
    [TestFixture(/* testWithHost = */ true)]
    [TestFixture(/* testWithHost = */ false)]
```